src/content/en/fundamentals/engage-and-retain/push-notifications/index.markdown

```diff
@@ -1,24 +1,77 @@
 ---
 layout: shared/narrow-pages-list
-title: "Push Notifications"
-description: "You may already know how to implement push notifications. Are you using them well? Could you be using them better?"
-published_on: 2016-01-05
-updated_on: 2016-01-05
+title: "Push Notifications: Timely, Relevant, and Precise"
+description: "Push notifications are one of the most valuable capabilities of native apps, and this capability is now available on the web. To get the most out of them, notifications need to be timely, precise, and relevant."
+published_on: 2016-07-01
+updated_on: 2016-07-01
 order: 1
-translation_priority: 0
+translation_priority: 1
 authors:
   - josephmedley
 ---
 
-<p class="intro">
-  You may already know <a href="/web/fundamentals/primers/push-notifications">how to implement push notifications</a>. Are you using them well? Could you be using them better? 
+<p class="intro">Don't annoy your users, or you'll lose access to them
```

Comment:
I'd love to see a screenshot here, something similar to the attached (but not the attached). Let me know if you want a hand creating it. Pri 2, not a requirement.

![joe](https://cloud.githubusercontent.com/assets/718038/16420296/ca5dc45c-3d1e-11e6-8dd5-bb4698971e20.jpg)